code/game/objects/items/grenades/chem_grenade.dm

```diff
@@ -27,6 +27,19 @@
 /obj/item/grenade/chem_grenade/examine(mob/user)
 	display_timer = (stage == READY && !nadeassembly)	//show/hide the timer based on assembly state
 	..()
+	if(user.can_see_reagents())
+		var/count = 0
+		if(beakers.len)
+			to_chat(user, "<span class='notice'>You scan the grenade and detect the following reagents:</span>")
+			for(var/obj/item/reagent_containers/glass/G in beakers)
+				count++
+				for(var/datum/reagent/R in G.reagents.reagent_list)
+					if(count == 1)

```

Comment:
recommend that you generalize this to make use of the `\th` macro like `[count]\th`, which will produce output `1st`, `2nd` etc.

Not a strict requirement, but it'd look nicer (in code) and support >2 beakers per nade

Reply:
I have never heard of this \th macro, is there any guideline or doc to read?

I don't think a grenade with >2 beakers is even possible in the code, but you're right that generalizations are always nice :)

Reply:
Changes done :)